### DISCUSSION TITLE
希望 Table 支持 dataIndex 泛型约束

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#48023] [General]
同 Form 的 https://ant-design.antgroup.com/components/form-cn#components-form-demo-basic

**Comment by yoyo837** [comment]
是不是就是这个 https://github.com/react-component/table/pull/1090 ?

**Reply by crazyair** [reply]
https://github.com/react-component/field-form/blob/master/src/interface.ts#L6

**Comment by crazyair** [comment]
https://github.com/react-component/table/pull/1091
